<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditRWParameterClass</class>
 <widget class="QMainWindow" name="EditRWParameterClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>610</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EditRWParameter</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>631</width>
      <height>561</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="Global">
     <attribute name="title">
      <string>Global</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>6</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="text">
       <string>File Name</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditGlobalParam">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>6</y>
        <width>501</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QFrame" name="frameRWParam">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>32</y>
        <width>611</width>
        <height>501</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Device">
     <attribute name="title">
      <string>Device</string>
     </attribute>
     <widget class="QFrame" name="frameFormDevice">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>611</width>
        <height>521</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QFrame" name="frameBottom">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>560</y>
      <width>631</width>
      <height>51</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="SaveButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
    <widget class="QPushButton" name="CancelButton">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>10</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
    <widget class="QPushButton" name="UpdateDefaultButton">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>10</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Overwrite</string>
     </property>
    </widget>
    <widget class="QPushButton" name="LoadButton">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>10</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Load</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
